msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/IBroker.java

```diff
@@ -44,24 +44,29 @@ default void removeAccount(PublicClientApplication application, IAccount account
         throw new MsalClientException("Broker implementation missing", AuthenticationErrorCode.MISSING_BROKER);
     }
 
+    /**
+     * Returns whether a broker is available and ready to use on this machine, allowing the use of the methods
+     * in this interface and other broker-only features in MSAL Java
+     */
     default boolean isBrokerAvailable() {
         throw new MsalClientException("Broker implementation missing", AuthenticationErrorCode.MISSING_BROKER);
     }
 
     /**
      * MSAL Java's AuthenticationResult requires several package-private classes that a broker implementation can't access,
-     *  so this helper method can be used to create AuthenticationResults from within the MSAL Java package
+     * so this helper method can be used to create AuthenticationResults from within the MSAL Java package
      */
     default IAuthenticationResult parseBrokerAuthResult(String authority, String idToken, String accessToken,
-                                                            String accountId, String clientInfo,
-                                                            long accessTokenExpirationTime) {
+                                                        String accountId, String clientInfo,
+                                                        long accessTokenExpirationTime,
+                                                        boolean isPopAuthorization) {
 
-        AuthenticationResult.AuthenticationResultBuilder builder =  AuthenticationResult.builder();
+        AuthenticationResult.AuthenticationResultBuilder builder = AuthenticationResult.builder();
 
         try {
             if (idToken != null) {
                 builder.idToken(idToken);
-                if (accountId!= null) {
+                if (accountId != null) {
                     String idTokenJson =
                             JWTParser.parse(idToken).getParsedParts()[1].decodeToString();
                     //TODO: need to figure out if 'policy' field is relevant for brokers
```

Comment:
No TODO in public code please. I guess the answer to this is no, but please open GH issues for tracking.

Reply:
Good catch, this was a leftover from the initial development. Removed in the latest commit.